app/assets/stylesheets/partials/_forms__mixin.scss

```diff
@@ -6,4 +6,9 @@
   .form-control-1-3.form-control-1-3 {
     width: 33%;
   }
+
+  .form-control-2-1.form-control-2-1 {
+    width: 150%;
```

Comment:
This is going to set the width wider than the page?

Reply:
Removed and added id specific (#country) width